app/components/concept-page/content.hbs

```diff
@@ -60,6 +60,17 @@
               </div>
             {{/animated-if}}
           </div>
+
+          <div class="py-12 border-t" {{did-insert this.handleShareConceptContainerInserted}} data-test-share-concept-container>
+            <div class="flex flex-col items-start gap-4 border rounded shadow-sm p-4">
+              <span class="flex gap-2 items-baseline">
```

Comment:
Alignment here looks off: 

<img width="542" alt="Screenshot 2024-02-15 at 21 09 59" src="https://github.com/codecrafters-io/frontend/assets/3893573/13861e44-adce-4d22-a749-68b8780fba2f">

Reply:
Let's do this instead?

```suggestion
              <span class="flex gap-2 items-center text-gray-700 text-sm">
```

<img width="456" alt="Screenshot 2024-02-15 at 21 12 18" src="https://github.com/codecrafters-io/frontend/assets/3893573/3ae2c8c7-f08e-4412-95c7-0ceab19d6455">

<img width="843" alt="Screenshot 2024-02-15 at 21 12 23" src="https://github.com/codecrafters-io/frontend/assets/3893573/e024f790-a42c-4348-88fe-e699601f5de4">